ecs/task-definition.json

```diff
@@ -2,8 +2,12 @@
   "containerDefinitions": [
     {
       "name": "server",
-      "cpu": 512,
+      "cpu": 1024,
       "command": ["npm", "run", "start"],
+      "environment": [{
+        "name": "NODE_OPTIONS",
+        "value": "--max-old-space-size=3072"
```

Comment:
Just a note for the future: Node's default is 2GB.

So here we allow `server` to use 3GB, `worker` to use 2GB of heap in Node, then `dd-agent` to use something 😅 These total to more than the 5GB cap set at the task level, but we don't think there's a real chance everything will max out at the same time so we are going with this config for now.